modules/programs/neomutt.nix

```diff
@@ -154,11 +156,16 @@ let
     set sidebar_format = '${cfg.sidebar.format}'
   '';
 
-  bindSection = concatMapStringsSep "\n"
-    (bind: ''bind ${bind.map} ${bind.key} "${bind.action}"'') cfg.binds;
+  genBindMapper = bindType:
+    concatMapStringsSep "\n" (bind:
+      ''
+        ${bindType} ${
+          concatStringsSep "," (toList bind.map)
```

Comment:
what if the list is empty ?

Reply:
I added an assertion in d1b06a6 to error if it is empty.